tools/lib/provision.py

```diff
@@ -385,6 +385,9 @@ def main(options):
     # Install shellcheck.
     run_as_root(["tools/setup/install-shellcheck"])
 
+    # Install sgrep.
+    run_as_root(["tools/setup/install-sgrep"])
```

Comment:
The `PROVISION_VERSION` variable in `version.py` will need to be bumped by a minor version so that `tools/lint` will suggest rerunning this.